package.json

```diff
@@ -12,7 +12,9 @@
     "test": "cross-env NODE_ENV=test nyc --reporter=text mocha  --require @babel/register --require @babel/polyfill  ./src/tests/*.js --timeout 30000 --exit",
     "down": "cross-env NODE_ENV=test node_modules/.bin/sequelize db:migrate:undo",
     "migrate": "cross-env NODE_ENV=test node_modules/.bin/sequelize db:migrate",
-    "seed": "node_modules/.bin/sequelize db:seed:all"
+    "seed": "node_modules/.bin/sequelize db:seed:all",
```

Comment:
remove the additional scripts. Looks like @Malaba6 added these as well

Reply:
Okay

Reply:
Excuse me... Which additional scripts? It's only the seed script. One is out and other is in?

Reply:
`downdev` and `migratedev`

Reply:
I use them for creating and removing tables without PG ADMIN.

Reply:
Okay. I will remove them!